src/profile-logic/marker-data.js

```diff
@@ -1176,7 +1224,8 @@ export function filterMarkerIndexesToRange(
   return filterMarkerIndexes(
     getMarker,
     markerIndexes,
-    marker => marker.start < rangeEnd && marker.start + marker.dur >= rangeStart
+    marker =>
+      marker.start <= rangeEnd && marker.start + marker.dur >= rangeStart
```

Comment:
It was hard for me to find this change in the PR and would be good to be separated to a new commit as you said in the chat. So feel free to make it a new commit if you want :)

Reply:
I'm going to leave this one here, since it's a bit of a pain at this point to rewrite the commits, and mark this comment as resolved. I do agree this would have been better as a separate commit.